assets/src/edit-story/components/richText/customExport.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { stateToHTML } from 'draft-js-export-html';
+
+/**
+ * Internal dependencies
+ */
+import generatePatternStyles from '../../utils/generatePatternStyles';
+import {
+  ITALIC,
+  UNDERLINE,
+  styleToWeight,
+  styleToLetterSpacing,
+  styleToColor,
+} from './customConstants';
+
+function inlineStyleFn(styles) {
+  const inline = styles.toArray().reduce(
+    ({ classes, css }, style) => {
+      // Italic
+      if (style === ITALIC) {
+        return {
+          classes: [...classes, 'italic'],
```

Comment:
They're only for re-parsing the content back in to the proper style names. It's easy to pattern match on the classes rather than the styling, was my thinking.

Reply:
Ok. If it's noticeable easier - sure. But `style` is pretty easy to interrogate as well, no? E.g. both `classList` and `style` can be cycled through easily, and both can also do membership checks just as easily.

Reply:
I've removed classes in refactor - using only styles now. See e.g weight [import](https://github.com/google/web-stories-wp/pull/1323/files#diff-c062d7168620373d6b39280d2038e7b8R40-R49) and [export/display](https://github.com/google/web-stories-wp/pull/1323/files#diff-c062d7168620373d6b39280d2038e7b8R51-R61).